components/slider/index.css

```diff
@@ -184,43 +214,34 @@ governing permissions and limitations under the License.
   }
 
   .spectrum-Slider-track {
-    &:before {
-      border-start-start-radius: 0;
-      border-end-start-radius: 0;
-      border-start-end-radius: 0;
-      border-end-end-radius: 0;
-    }
-
     &:first-of-type {
-      padding-inline: 0 var(--spectrum-slider-track-handleoffset);
-      inset-inline-start: var(--spectrum-slider-range-track-reset);
+      padding-inline: 0 var(--mod-slider-track-handleoffset, var(--spectrum-slider-track-handleoffset));
+      inset-inline-start: var(--mod-slider-range-track-reset, var(--spectrum-slider-range-track-reset));
       inset-inline-end: auto;
-      margin-inline-start: var(--spectrum-slider-track-margin-offset);
+      margin-inline-start: var(--mod-slider-track-margin-offset, var(--spectrum-slider-track-margin-offset));
+
       &:before {
-        border-start-start-radius: var(--spectrum-slider-m-track-border-radius);
-        border-end-start-radius: var(--spectrum-slider-m-track-border-radius);
+        border-start-start-radius: var(--mod-slider-track-corner-radius, var(--spectrum-slider-track-corner-radius));
+        border-end-start-radius: var(--mod-slider-track-corner-radius, var(--spectrum-slider-track-corner-radius));
         border-start-end-radius: 0;
         border-end-end-radius: 0;
+
+        /* border-start-start-radius: var(--mod-slider-track-corner-radius, var(--spectrum-slider-track-corner-radius));
```

Comment:
Is this content still needed or can it be cleaned up before merge?

Reply:
lets remove it.